apps/lab/src/app/page.tsx

```diff
@@ -1,12 +1,15 @@
 import { ViewWrapper } from './components';
+import { Button } from '@1xdev/get-random-number'
 
 import styles from './home.module.css';
 
 export default function Page() {
+
   return (
     <ViewWrapper>
       <div className={styles.main}>
         <h1 className={styles.hero}>Tech lab</h1>
+        <Button type='button' className={styles.btn}>Hello</Button>
```

Comment:
Now you need to verify using automation that the button is in the tech lab page.

Reply:
Tests are here: https://github.com/devxicans/platform/blob/main/apps/lab/__tests__/e2e/test.cy.tsx

Reply:
Alr